src/aiotk/stack.py

```diff
@@ -97,8 +114,17 @@ def __init__(self):
             if (etype, exc, tb) == (None, None, None):
                 return True
             raise exc
+        return None
+
+    @overload
+    async def enter_context(self, context: ContextManager) -> Any:
+        ...  # pragma: no cover
```

Comment:
**Note to self**: `coverage` does not seem to understand that these stubs for `mypy` are not actual code and we need to *explicitly* exclude each one.

Maybe we can somehow match this source line in our coverage exclusions?

Reply:
See #40 .